.github/workflows/unittest.yml

```diff
@@ -0,0 +1,42 @@
+name: Swift
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:    
+  test:
+    name: Test
```

Comment:
I think in the future we can split up this job to be a matrix job, so we can parallelize the testing on different devices better. For now, I thought we can keep it as it was on travis.